css-animations-2/Overview.bs

```diff
@@ -1448,15 +1448,69 @@ console.log(anim.playState); // Should be 'running'.
 
 </div>
 
-<h2 id="priv">
-  Privacy Considerations</h2>
+<h2 id="privacy">
+Privacy Considerations</h2>
 
-  No privacy concerns have been reported on this specification.
+<em>This section is not normative.</em>
 
-  <h2 id="sec">
-  Security Considerations</h2>
+As for security, the privacy considerations of this specification are limited
+because it does not allow Web content to do things that it could not do before.
 
-  No security concerns have been reported on this specification.
+This specification may provide additional mechanisms that help to determine
+characteristics of the user's hardware or software.
+However, ability to determine performance characteristics of the user's
+hardware or software is common to many Web technologies,
+and this specification does not introduce new capabilities.
+
+As described in [[#accessibility]],
+implementations may provide mitigations to help users with disabilities.
+These mitigations are likely to be detectable by Web content,
+which means that users who would benefit from these mitigations
+may face a tradeoff between keeping their disability private from
+the Web content or benefiting from the mitigation.
+
+<h2 id="sec">
+Security Considerations</h2>
+
+No security concerns have been reported on this specification.
+
+<h2 id="accessibility">
+Accessibility Considerations</h2>
+
+<em>This section is not normative.</em>
+
+<h3 id="accessibility-motion">
+Motion</h3>
+
+This specification provides declarative mechanisms for animations
+that previously needed to be done using script.
```

Comment:
This is a bit revealing of the spec history. It could also be that WAAPI came after css animations, but it doesn't change the content of the section. Not sure if there's some more neutral wording. This is also ok as is